src/field/pokemon.ts

```diff
@@ -3581,26 +3554,18 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     return false;
   }
 
-  /** @overload */
-  getTag(tagType: BattlerTagType): BattlerTag | nil;
-
-  /** @overload */
-  getTag<T extends BattlerTag>(tagType: AbstractConstructor<T>): T | nil;
-
-  getTag(tagType: BattlerTagType | AbstractConstructor<BattlerTag>): BattlerTag | nil {
+  getTag<T extends BattlerTag>(...tagTypes: BattlerTagType[]): T | nil {
```

Comment:
```suggestion
  getTag<T extends BattlerTag = BattlerTag>(...tagTypes: BattlerTagType[]): T | nil {
```
Is this intentionally different from `findTag()`?

Reply:
negative